docker-bake.hcl

```diff
@@ -0,0 +1,111 @@
+# VARIABLES
+
+variable "BUILD_TAG" {
+  default = "latest"
+}
+
+variable "SEMVER" {
+  default = "v0.0.0"
+}
+
+variable "GITCOMMIT" {
+  default = "dev"
+}
+
+variable "GITDATE" {
+  default = "0"
+}
+
+# GROUPS
+
+group "default" {
+  targets = ["all"]
+}
+
+group "all" {
+  targets = ["node-group", "disperser-group"]
+}
+
+group "node-group" {
+  targets = ["node", "nodeplugin"]
+}
+
+group "disperser-group" {
+  targets = ["batcher", "disperser", "encoder", "retriever", "churner"]
+}
+
+group "node-group-release" {
+  targets = ["node-release", "nodeplugin-release"]
+}
+
+# DISPERSER TARGETS
+
+target "batcher" {
+  context    = "."
+  dockerfile = "./Dockerfile"
+  target     = "batcher"
+  tags       = ["ghcr.io/layr-labs/eigenda/batcher:${BUILD_TAG}"]
+}
+
+target "disperser" {
+  context    = "."
+  dockerfile = "./Dockerfile"
+  target     = "apiserver"
+  tags       = ["ghcr.io/layr-labs/eigenda/disperser:${BUILD_TAG}"]
+}
+
+target "encoder" {
+  context    = "."
+  dockerfile = "./Dockerfile"
+  target     = "encoder"
+  tags       = ["ghcr.io/layr-labs/eigenda/encoder:${BUILD_TAG}"]
+}
+
+target "retriever" {
+  context    = "."
+  dockerfile = "./Dockerfile"
+  target     = "retriever"
+  tags       = ["ghcr.io/layr-labs/eigenda/retriever:${BUILD_TAG}"]
+}
+
+target "churner" {
+  context    = "."
+  dockerfile = "./Dockerfile"
+  target     = "churner"
+  tags       = ["ghcr.io/layr-labs/eigenda/churner:${BUILD_TAG}"]
+}
+
+# NODE TARGETS
+
+target "node" {
+  context    = "."
+  dockerfile = "./Dockerfile"
+  target     = "node"
+  tags       = ["ghcr.io/layr-labs/eigenda/node:${BUILD_TAG}"]
+  args = {
+    SEMVER    = "${SEMVER}"
+    GITCOMMIT = "${GITCOMMIT}"
+    GITDATE   = "${GITDATE}"
+  }
+}
+
+target "nodeplugin" {
+  context    = "."
+  dockerfile = "./Dockerfile"
+  target     = "nodeplugin"
+  tags       = ["ghcr.io/layr-labs/eigenda/nodeplugin:${BUILD_TAG}"]
+}
+
+# RELEASE TARGETS
+
+target "_release" {
```

Comment:
seems like it's not private, since you can still print it with `docker buildx bake _release --print`
Just indicative that it's not meant to be a real target.